### ISSUE TITLE
404 error on automatic api call

### ISSUE THREAD

**Issue by ciriousjoker** [chargebee/chargebee-js-wrappers#83]
```
GET https://[projectname]-test.chargebeestatic.com/api/internal/1646691300/retrieve_js_info 404 breadcrumbs.js:375
```

An incorrect public key would throw 401, so that's not it. It's also not because we're running a test instance because the sample project is also a test site.

### It works with the sample project
```ts
{
  site: 'honeycomics-v3-test',
  publishableKey: 'test_qoH22RugUvm5IcxoqUD5Svdcu9mX5figf'
}
```

_(We're using Angular if that matters)_

**Comment by nnmrts** [comment]
Encountering the same issue, any idea @cb-dinesh? And did you somehow find a workaround @ciriousjoker?

**Comment by ciriousjoker** [comment]
Nope. I've contacted support and got back a "will look at this", but nothing actually came out of it. No idea what this is, but it seems like it's not causing any issues so we just ignored it

**Comment by ciriousjoker** [comment]
@nnmrts Nope. I've contacted support and got back a "will look at this", but nothing actually came out of it. No idea what this is, but it seems like it's not causing any issues so we just ignored it

**Comment by nnmrts** [comment]
@ciriousjoker Thanks for the answer, I see...

 One "problem" is that several analysis tools will scream at you because there's a 404.
Another problem is that this request takes relatively long, blocking the whole site for like 200-500ms, which is fine for huge sites which load that long anyway. But my cache-optimized site loads instantly without Chargebee, but with it there's a short flicker that's quite annoying...

I found one other website on the internet that loads this `retrieve_js_info` successfully: https://linkvertise.com (open this site with a good adblocker like uBlock Origin)

Their JS is obfuscated so it's hard to see what they are doing right...

**Comment by MadaShindeInai** [comment]
Tried **test.chargebe.com** instead **test.chargebeestatic.com** , and request was successful. But i am not able to fix it somehow as it is part of external script. 
Also suggested Drop-In setup do not work for me. Iframe simply do not appear/opens.

**Comment by MadaShindeInai** [comment]
![Screenshot 2022-08-25 at 13 15 24](https://user-images.githubusercontent.com/50995397/186650341-5825676c-844f-41a9-ab9d-b84726de8c7c.png)

**Comment by MadaShindeInai** [comment]
UPD. Looks like the main issue fixed.
This errors still occur.
![Screenshot 2022-08-30 at 14 01 09](https://user-images.githubusercontent.com/50995397/187431431-be2cf506-1dd5-470b-814b-2c42015cf564.png)
